src/main/java/teammates/common/datatransfer/InstructorPrivileges.java

```diff
@@ -10,6 +10,75 @@
  * Representation of instructor privileges. Store the privileges of the instructor
  */
 public final class InstructorPrivileges {
+
+    private static final HashMap<String, Boolean> coownerPrivileges = new HashMap<>();
+    private static final HashMap<String, Boolean> managerPrivileges = new HashMap<>();
```

Comment:
Personally, I feel like this is pretty large change and should have been done in a different issue...